test/unit/specs/components/WwwFrame/TheHeader.spec.js

```diff
@@ -21,6 +21,7 @@ describe('TheHeader', () => {
 					apollo: {
 						readFragment: () => {},
 						query: () => Promise.resolve({}),
+						readQuery: () => Promise.resolve({}),
```

Comment:
I dont think readQuery returns a promise, since it only reads from the cache I think this should be `readQuery: () => {},`

Reply:
Done!